### ISSUE TITLE
Can we hide this share screen popup

### ISSUE THREAD

**Issue by Chandand76** [muaz-khan/RecordRTC#645]
Hello,

Its a good solution for screen capture with audio. 
I am using getDisplayMedia to capture screen, It will always show share screen popup window.

Also, i have used canvas recording, but for my application their are iframes and some other things and that doesn't suit me.

**Now Concerns is** :- 
Can we hide this share screen popup?
or is their any other way where this popup window will not show.

![image](https://user-images.githubusercontent.com/61581640/87756651-eab0f700-c826-11ea-9e86-906bbe9e6c20.png)

**Comment by PeteDevoy** [comment]
No, it is part of the browser.  Why would Google let their browser be used to leak private data without the user's consent?

As I was writing that I realised the irony of it but the answer is still "no" so I suggest you close this issue so as not to waste the time of the project author.

**Comment by usmnoor** [comment]
@PeteDevoy and how about changing somehow the elements on this popup window? When we include audio: true as a media constrain on getDisplayMedia(), the popup window appears with a "Share Audio" checkbox which is unchecked by default. Can we change it's state to checked by default when this popup window is opened?
